ta/secure_key_services/src/processing_ec.c

```diff
@@ -1083,7 +1083,14 @@ uint32_t sks2tee_ecdh_param_pub(struct sks_attribute_head *proc_params,
 
 	*pub_size = temp;
 
-	return serialargs_get_ptr(&args, pub_data, temp);
+	/* Skip DER tag */
+	rv = serialargs_get(&args, &temp, sizeof(uint8_t));
+	if (rv || temp != 0x04)
+		return rv;
```

Comment:
this will return with success if rv == SUCCESS and tmp != 0x04.

Reply:
right that is what it is supposed to do